src/map/atcommand.cpp

```diff
@@ -4030,15 +4030,33 @@ ATCE atcommand_character_storage_list(Session *s, dumb_ptr<map_session_data> sd,
 }
 
 static
+ATCE atcommand_pvp(Session *s, dumb_ptr<map_session_data> sd,
+        ZString)
+{
+    if (sd->pvp_timer)
+        return ATCE::OKAY;
+
+    sd->state.pvpon = !sd->state.pvpon;
+    pc_setpvptimer(sd, battle_config.player_pvp_time);
+
+    if (sd->state.pvpon)
+        clif_displaymessage(s, "PvP: On"_s);
```

Comment:
This should be sent to general tab. Please make it say "##3PvP : ##BOn" or something similar. The " : " will make it show in general tab, the ##3 will give it color and ##B will make it bold.